docs/syntax_and_semantics/enum.md

```diff
@@ -1,5 +1,7 @@
 # Enums
 
+This page is for [Crystal enums](https://crystal-lang.org/api/Enum.html). For C enums, see [C bindings enum](c_bindings/enum.md).
```

Comment:
Maybe this could be put into an admonition?

```suggestion
!!! note
    This page is for [Crystal enums](https://crystal-lang.org/api/Enum.html). For C enums, see [C bindings enum](c_bindings/enum.md).
```